sidecar/sidecar.go

```diff
@@ -110,8 +110,7 @@ func Run(conf config.Config) error {
 			return err
 		}
 
-		// TODO: Log failure?
-		go appSupervisor.DoLogManagement("/tmp/filebeat.yml")
+		appSupervisor.AddProcess([]string{"filebeat", "-c", "/tmp/filebeat.yml"}, []string{"GODEBUG=netdns=go"})
```

Comment:
@rshriram are we getting rid of the command arg log flag completely then?  How would someone supervise an app and log?  Two `apps` provided in the yaml file, one for logging and one for the app registered by service discovery?  Because if the logger is always a `helper` command, how you log in this case?